parts/k8s/kubernetesbase.t

```diff
@@ -16,11 +16,6 @@
       {{range $index, $agent := .AgentPoolProfiles}}
           "{{.Name}}Index": {{$index}},
           {{template "k8s/kubernetesagentvars.t" .}}
-          {{if IsNSeriesSKU .}}
-            {{if IsNVIDIADevicePluginEnabled}}
-            "registerWithGpuTaints": "nvidia.com/gpu=true:NoSchedule",
```

Comment:
needs to be removed in equivalent https://github.com/Azure/aks-engine/blob/master/parts/k8s/kubernetesbase.t#L21

Reply:
https://github.com/Azure/aks-engine/blob/master/pkg/engine/armvariables.go#L31 is the right link

Reply:
thanks! done